index.js

```diff
@@ -32,7 +32,8 @@ class ServerlessSecretBaker {
       "after:invoke:local:invoke": this.cleanupPackageSecrets.bind(this),
     };
 
-    const shouldCleanup = options["secret-baker-cleanup"] != false;
+    const params = this.readParams(options.param || []);
```

Comment:
Where is this `readParams` function defined?

Reply:
Sorry, other work got in the way and I still have local additions here that aren't pushed. I'll try to carve out some time early next week to complete this!